src/client/feed/feedReducer.js

```diff
@@ -0,0 +1,12 @@
+import { UPDATE_FEED_RESULTS } from '../search/SearchActions';
+
+const feed = (state = { 'posts': [] }, action) => {
+  switch (action.type) {
+    case UPDATE_FEED_RESULTS:

```

Comment:
I was talking about the constant, yes - sorry for the confusion haha

I would say that this (the case) isn't a part of the search specifically though, what if we wanted to say, after 1 minute of no input, refresh the feed with the latest results? That would be outside the scope of the searcher